pkgs/top-level/aliases.nix

```diff
@@ -411,6 +411,9 @@ mapAliases ({
   govendor = throw "'govendor' has been removed as it is no longer maintained upstream, please use Go modules instead"; # Added 2023-12-26
   gometer = throw "gometer has been removed from nixpkgs because goLance stopped offering Linux support"; # Added 2023-02-10
   gprbuild-boot = gnatPackages.gprbuild-boot; # Added 2024-02-25;
+
+  gpt4all-chat = throw "gpt4all-chat has been renamed to gpt4all"; # Added 2024-02-27
```

Comment:
I think you just alias first, especially in a project that doesn't have on-disk state. See https://github.com/NixOS/nixpkgs/commit/0c634976411118a2aace4fb6fe70cb398b73bc9a which gave about three years of aliasing prior to throwing.

Reply:
Hmmm. I thought it was older than that, but I'm just wrong. Let's roll with this version.